main.py

```diff
@@ -21,7 +21,6 @@ def main():
             print_error("Invalid Instructor ID. Try again. (q for quit)")
             continue
 
-        
```

Comment:
In main.py, there are a lot of unnecessary changes. Please remove those changes if you didn't intentionally change those lines (from lines 24 - 124)